scripts/reporting/batch_resolve_redcap_update_summary_scores.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python
+##
+##  See COPYING file distributed along with the ncanda-data-integration package
+##  for the copyright and license terms
+##
+
+import argparse
+import pandas as pd
+from pathlib import Path
+import sibispy
+import sibispy.cli
+from sibispy import sibislogger as slog
+import sys
+from typing import Sequence, List, Dict, Mapping, Union
+import yaml
+
+import github
+import pdb
+
+import batch_script_utils as utils
+
+def run_batch(verbose):
+    script_path = "/sibis-software/python-packages/sibispy/cmds/"
+    events = ["Baseline", "1y", "2y", "3y", "4y", "5y", "6y"]
```

Comment:
As in other PR - you'll need more years.

Reply:
The scripts should not (batch_resolve_redcap_update_summary_scores.py shouldn't even exist in the other branch) except for batch_script_utils, which will need to be merged since they're different in the two branches